app/src/main/res/layout/dialog_cancel_relation.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:orientation="vertical"
+              android:background="#f9f6e5"
+              android:layout_height="match_parent">
+
+    <TextView android:layout_width="match_parent"
+              android:layout_marginTop="8dp"
+              android:textSize="24sp"
+              android:layout_marginLeft="16dp"
+              android:textColor="#7e1b59"
+              android:textStyle="bold"
+              android:text="Cancel Relation !"
+              android:layout_height="wrap_content"/>
+
+
+    <TextView android:layout_width="match_parent"
+              android:layout_marginTop="16dp"
+              android:textSize="18sp"
+              android:layout_marginLeft="16dp"
+              android:textColor="#0a0909"
```

Comment:
Here also, define this color in **colors,xml** file and give a reference to that color here.